t/Util.pm

```diff
@@ -34,6 +34,33 @@ our $HttpConfig = <<_EOC_;
         local private_phases = require("kong.pdk.private.phases")
         local phases = private_phases.phases
 
+        -- This method executes 1 or more pdk methods twice: the first time with phase
+        -- checking deactivated, and the second time with phase checking activated.
+        -- Params:
+        -- * phase: the phase we want to test, i.e. "access"
+        -- * skip_fnlist controls a check: by default, this method
+        --         will check that the provided list of methods is "complete" - that
+        --         all the methods inside the `mod` (see below) are covered. Setting
+        --         `skip_fnlist` to `true` will skip that test (so the `mod` can have
+        --         methods that go untested)
+        --
+        -- This method also reads from 2 globals, which could be params instead:
```

Comment:
These are not parameters because they are defined in the test's `init` block and read by other blocks, and those only share globals. Since communication is happening via globals anyway, passing the parameter there would be just boilerplate added to every test. No point in adding a parameter if its value is always the same global.